### ISSUE TITLE
ipfs-cluster-ctl pin ls seems to be stuck forever

### ISSUE THREAD

**Issue by Katsuyu** [ipfs-cluster/ipfs-cluster#1782]
**Additional information:**
 - OS: Ubuntu 20.04
 - IPFS Cluster version: 1.0.0
 - Installation method: 2 node running ipfs-cluster in a docker-compose with following conf :
 ```
version: "2"
services:
  ipfs-cluster:
    cpu_quota: 90000
    mem_limit: 6gb
    restart: always
    image: ipfs/ipfs-cluster:v1.0.0
    environment:
      CLUSTER_PEERNAME: ipfs-cluster_${DEPLOYMENT_SERVER}
      CLUSTER_SECRET: ${CLUSTER_SECRET} # From shell variable if set
      CLUSTER_IPFSHTTP_NODEMULTIADDRESS: /dns4/ipfs/tcp/5001 # ipfs in the same docker network
      CLUSTER_CRDT_TRUSTEDPEERS: '*' # Trust all peers in Cluster
      CLUSTER_RESTAPI_HTTPLISTENMULTIADDRESS: /ip4/0.0.0.0/tcp/9094 # Expose API
      CLUSTER_MONITORPINGINTERVAL: 2s # Speed up peer discovery
      #%%PEER_ADDRESS%% // The peer address is set in the CI
    ports:
          # The cluster swarm port would need  to be exposed if this container
          # was to connect to cluster peers on other hosts.
          - "9096:9096" # Cluster IPFS Proxy endpoint
          - "9094:9094" # API port
    networks:
      ipfs_network:
        aliases:
          - ipfscluster_${DEPLOY_ENV}
    volumes:
      - ../data/ipfs-cluster_${DEPLOY_ENV}:/data/ipfs-cluster
networks:
  ipfs_network:
    external:
      name: ipfsbootstrap_${DEPLOY_ENV}_ipfs   
```

**Describe the bug:**

When going in the ipfs-cluster (only 6000 pins) container and running `ipfs-cluster-ctl pin ls`, the command is stuck forever, even when applying filter to reduce the list of pins, it keeps struggling.
Weirdly, on the contrary, when running `ipfs-cluster-ctl pin ls [CID]`, the result is instant and coherent.

Do you see any problem that could cause my issue ?

**Comment by hsanjuan** [comment]
Can you upgrade to the latest version (v1.0.4) first and test again?

For 6000 pins I would expect `pin ls` to be pretty fast, and pins should be streamed out immediately.

**Comment by Katsuyu** [comment]
Oh, my bad, the problem seems to has been solved between 1.0.0 and 1.0.4, tho I didn't see any reference to it in the changelog, I might have missed it !
Thanks !

**Comment by hsanjuan** [comment]
https://github.com/ipfs-cluster/ipfs-cluster/pull/1663 it was in the v1.0.1 changelog.
